mod-kb-ebsco/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -2566,36 +112,29 @@
 									"};",
 									"return new Promise(function(resolve, reject) {",
 									"  pm.sendRequest(echoGetRequest, (err, response) => {",
+									"      setTimeout(() => {",
 									"   if (!err) {",
-									"      var resp = response.text(); ",
-									"      var name = extractName(url);",
-									"      if(!checkVariableExist(name)){",
-									"        resp = replaceResponseRefWithName(resp);",
-									"        setEnvironmentVariable(name, resp);",
-									"      }",
-									"       traverse(JSON.parse(response.text()));",
-									"       resolve(resp);",
+									"       resolve(response.text());",
 									"   }else{",
-									"       reject(err);",
+									"       reject(err, echoGetRequest);",
 									"   }",
+									"      }, 5000);",
 									"  });",
```

Comment:
Great that this simplifies the schema retrieval process and it is more dynamic - Just curious on the timeout of 5000 - was there 1 specific schema retrieval which was taking longer that required the need for a timeout setting?

Reply:
it was added because of the connection errors while fetching schemas using Newman. 
Our schemas take max 1 sec on my machine to retrieve but decided to set 5 sec time to be sure that time is enough for getting schemas on any env. 
<img width="1680" alt="screen shot 2019-01-30 at 12 59 06 pm" src="https://user-images.githubusercontent.com/37537790/51976999-e9eba000-248e-11e9-9212-320fd4bb1a5a.png">

Reply:
Ok thank you for explanation!